tests/python/pants_test/backend/python/rules/BUILD

```diff
@@ -40,3 +40,16 @@ python_tests(
   timeout = 90,
 )
 
+python_tests(
+  name='python_fmt_integration',
+  source='test_python_fmt_integration.py',
+  dependencies=[
+    'tests/python/pants_test:int-test',
+    '//:pyproject',
+    'testprojects/src/python:unicode_directory',
+    'examples/src/python/example:hello_directory',
+  ],
+  tags={'integration'},
+  timeout = 90,
```

Comment:
How long does this take to run locally?

Reply:
Yeah, that's arbitrary. I thought 90 seconds was an OK default, but it should take less than 10 seconds. Just removed it and kept the default.